index.src.html

```diff
@@ -1692,6 +1711,11 @@
 
 dictionary SvcOutputMetadata {
   unsigned long temporalLayerId;
+  unsigned long spatiallayerId;
+  unsigned short frameNumber;
+  sequence<unsigned long> dependsOnIds;
```

Comment:
Why not dependencies like in RTCEncodedVideoFrameMetadata?

Reply:
`dependencies` in RTCEncodedVideoFrameMetadata is not defined but I'm assuming it is a sequence of `frameId` values.  `dependsOnIds` is a sequence of `frameNumber` values (though it seems odd that `frameNumber` is an unsigned short and `dependsOnIds` is a sequence of unsigned long).

Reply:
Shouldn't it be names dependsOnFrameNumbers then? 

Otherwise LGTM.